en/protocol/parameter.md

```diff
@@ -11,8 +68,187 @@ sequenceDiagram;
     Drone->>GCS: Send N parameters
     GCS->>GCS: Start receive timeout
     GCS->>Drone: Request dropped params with PARAM_REQUEST_READ
+{% endmermaid %}
+
+### Read Single Parameter
+
+A single parameter can be read by the [PARAM_REQUEST_READ](../messages/common.md#PARAM_REQUEST_READ) message.
+
 
+
+### Write Parameters
+
+As *QGroundControl* does not have its own list of the parameters on startup, before writing a parameter first the parameter list has to be read once. After that, parameters can be written individually by sending the key->value pair to the component. As QGroundControl keeps track of changed parameters, clicking the write button will only transmit those which have changed in value. The MAV **has to acknowledge the write operation** by emitting a [PARAM_REQUEST_READ](../messages/common.md#PARAM_VALUE)  value message with the newly written parameter value.
+
+{% mermaid %}
+sequenceDiagram;
+    participant GCS
+    participant Drone
+    GCS->>Drone: Semd parameter name and value
+    GCS->>GCS: Start timeout for receiving update value/ACK
+    Drone->>GCS: Send updated values
+    GCS->>Drone: If loss occurs: restart write transmission.
 {% endmermaid %}
 
+## Onboard Permanent Storage
+
+The parameter interface offers two buttons to load the parameters onboard from the permanent onboard storage into RAM and to save the current parameters in RAM to permanent storage. To write a new value from QGroundControl to permanent storage, first the value has to be transmitted and afterwards the write to ROM command has to be issued.
```

Comment:
This used to be the case but these days it's not used anymore. The `PREFLIGHT_STORAGE` command has disappeared from Firmware and QGC now.

Reply:
OK, I will delete this section.